Jenkinsfile

```diff
@@ -1003,28 +1003,28 @@ pipeline {
         } // stage('Test Hardware')
         stage ('Test Report') {
             parallel {
-                stage('Bullseye Report') {
+                stage('Bullseye Report on CentOS 8') {
```

Comment:
I cannot select line 1033 to add the comment there, but should these comments match, e.g.
```
} // stage('Bullseye Report on CentOS 8')
```

Reply:
Ideally.

Is it worth holding up a 2.0 blocker to fix these comments?  I will respin if you feel like they need fixing in this PR.

Reply:
@phender This PR is not as much of a long-pole as I understood it to be.  I will fix the comments and re-push.